apps/ws/src/socket/socket.module.ts

```diff
@@ -11,7 +11,7 @@ import { SharedModule } from '../shared/shared.module';
   exports: [WSGateway],
 })
 export class SocketModule implements OnModuleInit {
-  private readonly bullMqService: BullMqService;
+  public readonly bullMqService: BullMqService;
```

Comment:
Is `readonly` and I am looking to validate the right instantiation and values of the connection to Redis for this app. Therefore the change of visibility.